src/main/java/com/coreos/jetcd/internal/impl/LeaseImpl.java

```diff
@@ -313,13 +322,12 @@ private LeaseKeepAliveRequest newKeepAliveRequest(long leaseId) {
   }
 
 
-  private class KeepAliveListenerImpl implements KeepAliveListener {
+  private static class KeepAliveListenerImpl implements KeepAliveListener {
```

Comment:
@lburgazzoli I see. In my case, I realize that I don't need to access the inner class variable. I guess making the inner class static make sense for now.